tools/metadata/test/main.nf

```diff
@@ -19,7 +19,9 @@ include {fastq_metadata as meta_se; fastq_metadata as meta_pe;} from '../main.nf
 workflow {
     meta_se("$baseDir/../../../test_data/metadata/single_end_test.csv")
     meta_pe("$baseDir/../../../test_data/metadata/paired_end_test.csv")
+    meta_smartseq("$baseDir/../../../test_data/metadata/smartseq_test.csv")
 
     meta_se.out.metadata | view
     meta_pe.out.metadata | view
+    meta_smartseq.out | view
```

Comment:
could you add channel size checking to all the tests please? just to bring it in line with the model module